templates/common/_base/files/ofport-request.yaml

```diff
@@ -0,0 +1,114 @@
+        exit 0
+    fi
+    
+    # Get the port's slave-type. If this is not an ovs-bridge, then exit
+    PORT_OVS_SLAVE_TYPE=$(nmcli -t -f connection.slave-type conn show "$PORT_CONNECTION_UUID" | awk -F ':' '{print $NF}')
+    if [ "$PORT_OVS_SLAVE_TYPE" != "ovs-bridge" ]; then
+        exit 0
+    fi
+    
+    # Get the port's master = bridge name
+    BRIDGE_NAME=$(nmcli -t -f connection.master conn show "$PORT_CONNECTION_UUID" | awk -F ':' '{print $NF}')
+    # Limit this to br-ex and br-ex1 only. If one wanted to enable this for all OVS bridges,
+    # the condition would be: if [ "$BRIDGE_NAME" == "" ]; then
+    if [ "$BRIDGE_NAME" != "br-ex" ] && [ "$BRIDGE_NAME" != "br-ex1" ]; then
+        exit 0
+    fi
+    
+    CONFIGURATION_FILE=/run/ofport_requests.${BRIDGE_NAME}
+    
+    # Declare a new map. If CONFIGURATION_FILE exists, source entries from there
+    declare -A INTERFACES
+    if [ -f $CONFIGURATION_FILE ]; then
+        echo "Sourcing configuration file '${CONFIGURATION_FILE}' with contents: $(cat $CONFIGURATION_FILE)"
+        source $CONFIGURATION_FILE
+    fi
+    
+    # get_interface_ofport_request will return
+    # * either: the current ofport assignment for the port if no interface has claimed this ofport number, yet
+    # * or:     the lowest available free ofport number
+    function get_interface_ofport_request() {
+        # Build a map that only contains the currently reserved ofport_requests
+        declare -A ofport_requests
+        for interface_name in "${!INTERFACES[@]}"; do
+    	ofport_requests[${INTERFACES[$interface_name]}]=${INTERFACES[$interface_name]}
+        done
+    
+        # Get the current ofport number assignment
+        local current_ofport=$(ovs-vsctl get Interface "$INTERFACE_NAME" ofport)
+    
+        # If the current ofport number is still free, use it
+        if ! [ "${ofport_requests[$current_ofport]+a}" ]; then
+    	    echo $current_ofport
+    	    return
+        fi
+    
+        # If the current ofport number is not free, return the lowest free entry
+        i=0
+        for i in {1..65000}; do
+            if ! [ "${ofport_requests[$i]+a}" ]; then
+                echo $i
+                return
+            fi
+        done
+    
+        # if we still cannot find an ID, exit with an error
+        echo "Impossible to find an ofport ID for interface $INTERFACE_NAME" >&2
+        exit 1
+    }
+    
+    # If INTERFACES[INTERFACE_NAME] exists, use that value
+    # If INTERFACES[INTERFACE_NAME] does not exists, use get_interface_ofport_request to get a new value
+    if ! [ "${INTERFACES[$INTERFACE_NAME]+a}" ]; then
+    	INTERFACES[$INTERFACE_NAME]=$(get_interface_ofport_request)
+    fi
+    # Set ofport_request according to INTERFACES[INTERFACE_NAME]
+    ovs-vsctl set Interface "$INTERFACE_NAME" ofport_request=${INTERFACES[$INTERFACE_NAME]}
+    
+    # Save current state of INTERFACES to CONFIGURATION_FILE
+    declare -p INTERFACES > $CONFIGURATION_FILE
```

Comment:
Would this fail with `file exists` if it exists?

Reply:
It should not fail, it writes the state to the file which is what we want.
I'll test what happens if this is a directory or if the file is immutable, just to make sure if this will crash or not.

Reply:
This is what happens to me:
```
/tmp
❯ touch test                                                

/tmp
❯ declare -A INTERFACES

/tmp
❯ declare -p INTERFACES > test
zsh: file exists: test

```

Reply:
Lengthy comment here, but here is it from my test - I created a directoy in place of the file, and whereas the script fails, NM and the connection do not, so we should be o.k.:
~~~
[root@rhel84 pre-up.d]# journalctl -f | grep nm-dispa &
[1] 5634
[root@rhel84 pre-up.d]# systemctl restart NetworkManager
[root@rhel84 pre-up.d]# Feb 08 11:18:01 rhel84 dbus-daemon[774]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.336' (uid=0 pid=5072 comm="/usr/sbin/NetworkManager --no-daemon " label="system_u:system_r:NetworkManager_t:s0")
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=bond1
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="bond1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80 == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + PORT=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="1cc55059-da8f-4838-873c-6458e4c2682b" || $3=="1cc55059-da8f-4838-873c-6458e4c2682b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface bond1 ofport
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ local current_ofport=
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 82: ofport_requests: bad array subscript
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ echo
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface bond1 ofport_request=
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: req:35 'pre-up' [bond1], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth6
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth6" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' 23969750-49ef-4145-99c3-6af53b54f4a4 == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth7
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth7" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' d20b59c8-99c8-4521-980f-9422b1682ab4 == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth0
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth0" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' a63944ec-3426-3ec1-b485-5b1603aadf3d == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' '' '!=' ovs-port ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth4
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth4" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: + '[' 261b0374-1446-4f19-a2dd-5494b70212fe == '' ']'
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:02 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT=bond0
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' bond0 == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth4 ofport
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ local current_ofport=43
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ echo 43
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=43
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth4 ofport_request=43
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: req:44 'pre-up' [eth4], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth3
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth3" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' 195a5964-2eff-4ead-8bda-0b071739e9d6 == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d" || $3=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex2
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' br-ex2 '!=' br-ex ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' br-ex2 '!=' br-ex1 ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth5
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth5" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' 5d0fc00d-17e0-471a-b0a9-4bd851876d83 == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT=bond0
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' bond0 == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:03 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth5 ofport
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ local current_ofport=47
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ echo 47
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=47
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth5 ofport_request=47
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: req:57 'pre-up' [eth5], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth1
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ec425a9c-9128-4d6e-a367-8cb0493f688f == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="47871482-b875-4ba0-bb26-5e213135583f" || $3=="47871482-b875-4ba0-bb26-5e213135583f") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth1 ofport
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ local current_ofport=46
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ echo 46
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=46
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth1 ofport_request=46
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: req:58 'pre-up' [eth1], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth2
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' c98f0470-7955-4195-bc0e-2174dc49085f == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11" || $3=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth2 ofport
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ local current_ofport=45
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ echo 45
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=45
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth2 ofport_request=45
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: req:59 'pre-up' [eth2], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=vlan10
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="vlan10" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 4dfb2bb4-45ed-4fff-b746-6432874fde5a == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b" || $3=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface vlan10 ofport
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ local current_ofport=44
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ echo 44
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=44
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface vlan10 ofport_request=44
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: req:60 'pre-up' [vlan10], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=br-ex2
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="br-ex2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' '' == '' ']'
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=br-ex
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:04 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:05 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:05 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="br-ex" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:05 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:05 rhel84 nm-dispatcher[5640]: + '[' '' == '' ']'
Feb 08 11:18:05 rhel84 nm-dispatcher[5640]: + exit 0

[root@rhel84 pre-up.d]# ovs-ofctl show br-ex
OFPT_FEATURES_REPLY (xid=0x2): dpid:0000caab9438e140
n_tables:254, n_buffers:0
capabilities: FLOW_STATS TABLE_STATS PORT_STATS QUEUE_STATS ARP_MATCH_IP
actions: output enqueue set_vlan_vid set_vlan_pcp strip_vlan mod_dl_src mod_dl_dst mod_nw_src mod_nw_dst mod_nw_tos mod_tp_src mod_tp_dst
 32(patch-to-br-int): addr:4e:62:7d:8c:d4:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 43(eth4): addr:52:54:00:df:9c:35
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 44(vlan10): addr:52:54:00:33:bc:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 45(eth2): addr:52:54:00:94:f8:58
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 46(eth1): addr:52:54:00:68:cf:ce
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 47(eth5): addr:52:54:00:15:ec:29
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 LOCAL(br-ex): addr:ca:ab:94:38:e1:40
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
OFPT_GET_CONFIG_REPLY (xid=0x4): frags=normal miss_send_len=0
[root@rhel84 pre-up.d]# systemctl restart NetworkManager
[root@rhel84 pre-up.d]# Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=bond1
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="bond1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80 == '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + PORT=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="1cc55059-da8f-4838-873c-6458e4c2682b" || $3=="1cc55059-da8f-4838-873c-6458e4c2682b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface bond1 ofport
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ local current_ofport=
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 82: ofport_requests: bad array subscript
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ echo
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface bond1 ofport_request=
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: req:111 'pre-up' [bond1], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth6
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth6" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' 23969750-49ef-4145-99c3-6af53b54f4a4 == '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth7
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth7" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: + '[' d20b59c8-99c8-4521-980f-9422b1682ab4 == '' ']'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:12 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth0
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth0" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' a63944ec-3426-3ec1-b485-5b1603aadf3d == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' '' '!=' ovs-port ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth3
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth3" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' 195a5964-2eff-4ead-8bda-0b071739e9d6 == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + PORT=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d" || $3=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex2
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' br-ex2 '!=' br-ex ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' br-ex2 '!=' br-ex1 ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth1
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' ec425a9c-9128-4d6e-a367-8cb0493f688f == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + PORT=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="47871482-b875-4ba0-bb26-5e213135583f" || $3=="47871482-b875-4ba0-bb26-5e213135583f") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth1 ofport
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ local current_ofport=49
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ echo 49
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=49
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth1 ofport_request=49
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: req:133 'pre-up' [eth1], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth5
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth5" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' 5d0fc00d-17e0-471a-b0a9-4bd851876d83 == '' ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:13 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT=bond0
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' bond0 == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth5 ofport
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ local current_ofport=50
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ echo 50
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=50
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth5 ofport_request=50
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: req:134 'pre-up' [eth5], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth2
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' c98f0470-7955-4195-bc0e-2174dc49085f == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11" || $3=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth2 ofport
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ local current_ofport=48
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ echo 48
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=48
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth2 ofport_request=48
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: req:135 'pre-up' [eth2], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=eth4
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="eth4" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 261b0374-1446-4f19-a2dd-5494b70212fe == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT=bond0
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' bond0 == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface eth4 ofport
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ local current_ofport=51
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ echo 51
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=51
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface eth4 ofport_request=51
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: req:136 'pre-up' [eth4], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=vlan10
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="vlan10" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 4dfb2bb4-45ed-4fff-b746-6432874fde5a == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if( ($1=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b" || $3=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + PORT_CONNECTION_UUID=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.slave-type conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:14 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f connection.master conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ awk -F : '{print $NF}'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + BRIDGE_NAME=br-ex
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + declare -A INTERFACES
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' '' ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ get_interface_ofport_request
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ declare -A ofport_requests
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: +++ ovs-vsctl get Interface vlan10 ofport
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ local current_ofport=52
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ '[' '' ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ echo 52
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ return
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + INTERFACES[$INTERFACE_NAME]=52
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + ovs-vsctl set Interface vlan10 ofport_request=52
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + declare -p INTERFACES
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 110: /run/ofport_requests.br-ex: Is a directory
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: req:137 'pre-up' [vlan10], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=br-ex2
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="br-ex2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' '' == '' ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + exit 0
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + INTERFACE_NAME=br-ex
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + OPERATION=pre-up
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ awk -F : '{if($1=="br-ex" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + '[' '' == '' ']'
Feb 08 11:18:15 rhel84 nm-dispatcher[5640]: + exit 0

[root@rhel84 pre-up.d]# nmcli conn
NAME                UUID                                  TYPE           DEVICE 
Wired connection 2  923b22f9-8cc8-32fa-aa0d-54cffb024215  ethernet       eth10  
Wired connection 3  2b1f9ef1-5fb3-35f5-9269-5ab7ef70f5c9  ethernet       eth8   
Wired connection 4  9432aa2c-d1aa-30a1-b852-970bc72b687d  ethernet       eth9   
Wired connection 1  a63944ec-3426-3ec1-b485-5b1603aadf3d  ethernet       eth0   
ovs-if-br-ex        74c03eca-f6a0-4409-b4e5-2698e0e0665a  ovs-interface  br-ex  
ovs-if-br-ex2       4f70d1f0-48f7-42fa-b4bd-d7f69e4477de  ovs-interface  br-ex2 
bond1               cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80  bond           bond1  
bond1-eth6          23969750-49ef-4145-99c3-6af53b54f4a4  ethernet       eth6   
bond1-eth7          d20b59c8-99c8-4521-980f-9422b1682ab4  ethernet       eth7   
bond1-ovsport       1cc55059-da8f-4838-873c-6458e4c2682b  ovs-port       bond1  
br-ex               9e0055c1-23e7-473c-8444-015d4aa11dac  ovs-bridge     br-ex  
br-ex2              34109401-5f47-44ca-9f3b-5fc1b37422a5  ovs-bridge     br-ex2 
ovs-if-eth1         ec425a9c-9128-4d6e-a367-8cb0493f688f  ethernet       eth1   
ovs-if-eth2         c98f0470-7955-4195-bc0e-2174dc49085f  ethernet       eth2   
ovs-if-eth3         195a5964-2eff-4ead-8bda-0b071739e9d6  ethernet       eth3   
ovs-port-br-ex      0d18b6a6-2fe5-4c44-b705-d2fc4301aafc  ovs-port       br-ex  
ovs-port-br-ex2     8ba11cc7-a222-40f5-86a4-c569a85f81f4  ovs-port       br-ex2 
ovs-port-eth1       47871482-b875-4ba0-bb26-5e213135583f  ovs-port       eth1   
ovs-port-eth2       5b9bdfeb-7238-45dc-b7af-714b9a12ab11  ovs-port       eth2   
ovs-port-eth3       2f4f0995-7ead-4ffe-873c-e1f544ce1b7d  ovs-port       eth3   
ovs-slave-bond0     676b62a4-63b8-4c85-8cba-c1a091616e19  ovs-port       bond0  
ovs-slave-eth4      261b0374-1446-4f19-a2dd-5494b70212fe  ethernet       eth4   
ovs-slave-eth5      5d0fc00d-17e0-471a-b0a9-4bd851876d83  ethernet       eth5   
vlan10              4dfb2bb4-45ed-4fff-b746-6432874fde5a  vlan           vlan10 
vlan10-ovsport      1e7cf87e-a6c0-467c-ae88-70fb0e0d701b  ovs-port       vlan10 
[root@rhel84 pre-up.d]# ovs-ofctl show br-ex
OFPT_FEATURES_REPLY (xid=0x2): dpid:0000caab9438e140
n_tables:254, n_buffers:0
capabilities: FLOW_STATS TABLE_STATS PORT_STATS QUEUE_STATS ARP_MATCH_IP
actions: output enqueue set_vlan_vid set_vlan_pcp strip_vlan mod_dl_src mod_dl_dst mod_nw_src mod_nw_dst mod_nw_tos mod_tp_src mod_tp_dst
 32(patch-to-br-int): addr:4e:62:7d:8c:d4:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 48(eth2): addr:52:54:00:94:f8:58
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 49(eth1): addr:52:54:00:68:cf:ce
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 50(eth5): addr:52:54:00:15:ec:29
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 51(eth4): addr:52:54:00:df:9c:35
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 52(vlan10): addr:52:54:00:33:bc:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 LOCAL(br-ex): addr:ca:ab:94:38:e1:40
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
OFPT_GET_CONFIG_REPLY (xid=0x4): frags=normal miss_send_len=0
[root@rhel84 pre-up.d]# rmdir /var/run/ofport_requests.br-ex/
[root@rhel84 pre-up.d]# systemctl restart NetworkManager
[root@rhel84 pre-up.d]# Feb 08 11:18:31 rhel84 dbus-daemon[774]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.442' (uid=0 pid=6228 comm="/usr/sbin/NetworkManager --no-daemon " label="system_u:system_r:NetworkManager_t:s0")
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=bond1
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="bond1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + '[' cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80 == '' ']'
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + PORT=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:31 rhel84 nm-dispatcher[6799]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="1cc55059-da8f-4838-873c-6458e4c2682b" || $3=="1cc55059-da8f-4838-873c-6458e4c2682b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + declare -A INTERFACES
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ get_interface_ofport_request
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ declare -A ofport_requests
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: +++ ovs-vsctl get Interface bond1 ofport
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ local current_ofport=
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 82: ofport_requests: bad array subscript
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ '[' '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ echo
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ return
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACES[$INTERFACE_NAME]=
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + ovs-vsctl set Interface bond1 ofport_request=
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: req:42 'pre-up' [bond1], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth6
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth6" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 23969750-49ef-4145-99c3-6af53b54f4a4 == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + exit 0
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth7
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth7" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' d20b59c8-99c8-4521-980f-9422b1682ab4 == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + exit 0
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth0
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth0" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' a63944ec-3426-3ec1-b485-5b1603aadf3d == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' '' '!=' ovs-port ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + exit 0
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth3
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth3" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 195a5964-2eff-4ead-8bda-0b071739e9d6 == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d" || $3=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex2
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' br-ex2 '!=' br-ex ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' br-ex2 '!=' br-ex1 ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + exit 0
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth1
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' ec425a9c-9128-4d6e-a367-8cb0493f688f == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="47871482-b875-4ba0-bb26-5e213135583f" || $3=="47871482-b875-4ba0-bb26-5e213135583f") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:32 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ get_interface_ofport_request
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A ofport_requests
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: +++ ovs-vsctl get Interface eth1 ofport
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ local current_ofport=55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ echo 55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ return
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACES[$INTERFACE_NAME]=55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + ovs-vsctl set Interface eth1 ofport_request=55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -p INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth4
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth4" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' 261b0374-1446-4f19-a2dd-5494b70212fe == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT=bond0
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' bond0 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth1]="55" )'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth1]="55" )
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + source /run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ INTERFACES=([eth1]="55")
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ get_interface_ofport_request
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A ofport_requests
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: +++ ovs-vsctl get Interface eth4 ofport
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ local current_ofport=57
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ echo 57
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ return
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACES[$INTERFACE_NAME]=57
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + ovs-vsctl set Interface eth4 ofport_request=57
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -p INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth2
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' c98f0470-7955-4195-bc0e-2174dc49085f == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11" || $3=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth1]="55" [eth4]="57" )'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth1]="55" [eth4]="57" )
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + source /run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ INTERFACES=([eth1]="55" [eth4]="57")
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ get_interface_ofport_request
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A ofport_requests
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=57
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: +++ ovs-vsctl get Interface eth2 ofport
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ local current_ofport=54
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ echo 54
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ return
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACES[$INTERFACE_NAME]=54
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + ovs-vsctl set Interface eth2 ofport_request=54
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -p INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=eth5
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="eth5" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' 5d0fc00d-17e0-471a-b0a9-4bd851876d83 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT=bond0
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' bond0 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth4]="57" )'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth4]="57" )
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + source /run/ofport_requests.br-ex
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth4]="57")
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A INTERFACES
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ get_interface_ofport_request
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ declare -A ofport_requests
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=54
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=55
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=57
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: +++ ovs-vsctl get Interface eth5 ofport
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ local current_ofport=56
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ '[' '' ']'
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ echo 56
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: ++ return
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + INTERFACES[$INTERFACE_NAME]=56
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + ovs-vsctl set Interface eth5 ofport_request=56
Feb 08 11:18:33 rhel84 nm-dispatcher[6799]: + declare -p INTERFACES
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=vlan10
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="vlan10" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' 4dfb2bb4-45ed-4fff-b746-6432874fde5a == '' ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + PORT=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if( ($1=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b" || $3=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + PORT_CONNECTION_UUID=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.slave-type conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f connection.master conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + BRIDGE_NAME=br-ex
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + declare -A INTERFACES
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" )'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" )
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + source /run/ofport_requests.br-ex
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57")
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ declare -A INTERFACES
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' '' ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ get_interface_ofport_request
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ declare -A ofport_requests
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=54
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=55
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=56
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ ofport_requests[${INTERFACES[$interface_name]}]=57
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: +++ ovs-vsctl get Interface vlan10 ofport
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ local current_ofport=53
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ '[' '' ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ echo 53
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ return
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + INTERFACES[$INTERFACE_NAME]=53
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + ovs-vsctl set Interface vlan10 ofport_request=53
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + declare -p INTERFACES
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + INTERFACE_NAME=br-ex2
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + OPERATION=pre-up
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: ++ awk -F : '{if($1=="br-ex2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + '[' '' == '' ']'
Feb 08 11:18:34 rhel84 nm-dispatcher[6799]: + exit 0

[root@rhel84 pre-up.d]# nmcli conn
NAME                UUID                                  TYPE           DEVICE 
Wired connection 2  923b22f9-8cc8-32fa-aa0d-54cffb024215  ethernet       eth10  
Wired connection 3  2b1f9ef1-5fb3-35f5-9269-5ab7ef70f5c9  ethernet       eth8   
Wired connection 4  9432aa2c-d1aa-30a1-b852-970bc72b687d  ethernet       eth9   
Wired connection 1  a63944ec-3426-3ec1-b485-5b1603aadf3d  ethernet       eth0   
ovs-if-br-ex2       4f70d1f0-48f7-42fa-b4bd-d7f69e4477de  ovs-interface  br-ex2 
bond1               cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80  bond           bond1  
bond1-eth6          23969750-49ef-4145-99c3-6af53b54f4a4  ethernet       eth6   
bond1-eth7          d20b59c8-99c8-4521-980f-9422b1682ab4  ethernet       eth7   
bond1-ovsport       1cc55059-da8f-4838-873c-6458e4c2682b  ovs-port       bond1  
br-ex               9e0055c1-23e7-473c-8444-015d4aa11dac  ovs-bridge     br-ex  
br-ex2              34109401-5f47-44ca-9f3b-5fc1b37422a5  ovs-bridge     br-ex2 
ovs-if-eth1         ec425a9c-9128-4d6e-a367-8cb0493f688f  ethernet       eth1   
ovs-if-eth2         c98f0470-7955-4195-bc0e-2174dc49085f  ethernet       eth2   
ovs-if-eth3         195a5964-2eff-4ead-8bda-0b071739e9d6  ethernet       eth3   
ovs-port-br-ex      0d18b6a6-2fe5-4c44-b705-d2fc4301aafc  ovs-port       br-ex  
ovs-port-br-ex2     8ba11cc7-a222-40f5-86a4-c569a85f81f4  ovs-port       br-ex2 
ovs-port-eth1       47871482-b875-4ba0-bb26-5e213135583f  ovs-port       eth1   
ovs-port-eth2       5b9bdfeb-7238-45dc-b7af-714b9a12ab11  ovs-port       eth2   
ovs-port-eth3       2f4f0995-7ead-4ffe-873c-e1f544ce1b7d  ovs-port       eth3   
ovs-slave-bond0     676b62a4-63b8-4c85-8cba-c1a091616e19  ovs-port       bond0  
ovs-slave-eth4      261b0374-1446-4f19-a2dd-5494b70212fe  ethernet       eth4   
ovs-slave-eth5      5d0fc00d-17e0-471a-b0a9-4bd851876d83  ethernet       eth5   
vlan10              4dfb2bb4-45ed-4fff-b746-6432874fde5a  vlan           vlan10 
vlan10-ovsport      1e7cf87e-a6c0-467c-ae88-70fb0e0d701b  ovs-port       vlan10 
ovs-if-br-ex        74c03eca-f6a0-4409-b4e5-2698e0e0665a  ovs-interface  --     
[root@rhel84 pre-up.d]# ovs-ofctl show br-ex
OFPT_FEATURES_REPLY (xid=0x2): dpid:0000caab9438e140
n_tables:254, n_buffers:0
capabilities: FLOW_STATS TABLE_STATS PORT_STATS QUEUE_STATS ARP_MATCH_IP
actions: output enqueue set_vlan_vid set_vlan_pcp strip_vlan mod_dl_src mod_dl_dst mod_nw_src mod_nw_dst mod_nw_tos mod_tp_src mod_tp_dst
 32(patch-to-br-int): addr:4e:62:7d:8c:d4:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 53(vlan10): addr:52:54:00:33:bc:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 54(eth2): addr:52:54:00:94:f8:58
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 55(eth1): addr:52:54:00:68:cf:ce
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 56(eth5): addr:52:54:00:15:ec:29
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 57(eth4): addr:52:54:00:df:9c:35
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
OFPT_GET_CONFIG_REPLY (xid=0x4): frags=normal miss_send_len=0
[root@rhel84 pre-up.d]# systemctl restart NetworkManager
[root@rhel84 pre-up.d]# Feb 08 11:18:47 rhel84 dbus-daemon[774]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.496' (uid=0 pid=6811 comm="/usr/sbin/NetworkManager --no-daemon " label="system_u:system_r:NetworkManager_t:s0")
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=bond1
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="bond1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80 == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show cd4c8622-8ccd-4a3e-b2a1-9b1d57667e80
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + PORT=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="1cc55059-da8f-4838-873c-6458e4c2682b" || $3=="1cc55059-da8f-4838-873c-6458e4c2682b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' 1cc55059-da8f-4838-873c-6458e4c2682b == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 1cc55059-da8f-4838-873c-6458e4c2682b
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + declare -A INTERFACES
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + source /run/ofport_requests.br-ex
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53")
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ declare -A INTERFACES
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ get_interface_ofport_request
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ declare -A ofport_requests
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ ofport_requests[${INTERFACES[$interface_name]}]=54
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ ofport_requests[${INTERFACES[$interface_name]}]=55
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ ofport_requests[${INTERFACES[$interface_name]}]=56
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ ofport_requests[${INTERFACES[$interface_name]}]=57
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ for interface_name in "${!INTERFACES[@]}"
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ ofport_requests[${INTERFACES[$interface_name]}]=53
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: +++ ovs-vsctl get Interface bond1 ofport
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ local current_ofport=
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: /etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh: line 82: ofport_requests: bad array subscript
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ '[' '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ echo
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ return
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACES[$INTERFACE_NAME]=
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + ovs-vsctl set Interface bond1 ofport_request=
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ovs-vsctl: no row "bond1" in table Interface
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: req:34 'pre-up' [bond1], "/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh": complete: failed with Script '/etc/NetworkManager/dispatcher.d/pre-up.d/10-ofport-request.sh' exited with status 1.
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth6
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth6" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' 23969750-49ef-4145-99c3-6af53b54f4a4 == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 23969750-49ef-4145-99c3-6af53b54f4a4
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + exit 0
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth7
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth7" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' d20b59c8-99c8-4521-980f-9422b1682ab4 == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show d20b59c8-99c8-4521-980f-9422b1682ab4
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=bond
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' bond '!=' ovs-port ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + exit 0
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth0
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth0" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' a63944ec-3426-3ec1-b485-5b1603aadf3d == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show a63944ec-3426-3ec1-b485-5b1603aadf3d
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' '' '!=' ovs-port ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + exit 0
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth4
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth4" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: + '[' 261b0374-1446-4f19-a2dd-5494b70212fe == '' ']'
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:48 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 261b0374-1446-4f19-a2dd-5494b70212fe
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT=bond0
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' bond0 == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + declare -A INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + source /run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53")
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ declare -A INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' a ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + ovs-vsctl set Interface eth4 ofport_request=57
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + declare -p INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth1
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth1" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ec425a9c-9128-4d6e-a367-8cb0493f688f == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show ec425a9c-9128-4d6e-a367-8cb0493f688f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="47871482-b875-4ba0-bb26-5e213135583f" || $3=="47871482-b875-4ba0-bb26-5e213135583f") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' 47871482-b875-4ba0-bb26-5e213135583f == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 47871482-b875-4ba0-bb26-5e213135583f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + declare -A INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + source /run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53")
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ declare -A INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' a ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + ovs-vsctl set Interface eth1 ofport_request=55
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + declare -p INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth2
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' c98f0470-7955-4195-bc0e-2174dc49085f == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show c98f0470-7955-4195-bc0e-2174dc49085f
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11" || $3=="5b9bdfeb-7238-45dc-b7af-714b9a12ab11") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' 5b9bdfeb-7238-45dc-b7af-714b9a12ab11 == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 5b9bdfeb-7238-45dc-b7af-714b9a12ab11
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + declare -A INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + source /run/ofport_requests.br-ex
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53")
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ declare -A INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' a ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + ovs-vsctl set Interface eth2 ofport_request=54
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + declare -p INTERFACES
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth5
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth5" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' 5d0fc00d-17e0-471a-b0a9-4bd851876d83 == '' ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 5d0fc00d-17e0-471a-b0a9-4bd851876d83
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + PORT=bond0
Feb 08 11:18:49 rhel84 nm-dispatcher[7391]: + '[' bond0 == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="bond0" || $3=="bond0") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 676b62a4-63b8-4c85-8cba-c1a091616e19 == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 676b62a4-63b8-4c85-8cba-c1a091616e19
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + declare -A INTERFACES
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + source /run/ofport_requests.br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53")
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ declare -A INTERFACES
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' a ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + ovs-vsctl set Interface eth5 ofport_request=56
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + declare -p INTERFACES
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=eth3
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="eth3" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 195a5964-2eff-4ead-8bda-0b071739e9d6 == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 195a5964-2eff-4ead-8bda-0b071739e9d6
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d" || $3=="2f4f0995-7ead-4ffe-873c-e1f544ce1b7d") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 2f4f0995-7ead-4ffe-873c-e1f544ce1b7d
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex2
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' br-ex2 '!=' br-ex ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' br-ex2 '!=' br-ex1 ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + exit 0
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=vlan10
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="vlan10" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 4dfb2bb4-45ed-4fff-b746-6432874fde5a == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_OVS_SLAVE_TYPE=ovs-port
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' ovs-port '!=' ovs-port ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 4dfb2bb4-45ed-4fff-b746-6432874fde5a
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if( ($1=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b" || $3=="1e7cf87e-a6c0-467c-ae88-70fb0e0d701b") && $2~/^ovs*/) print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT_CONNECTION_UUID=1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.slave-type conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + PORT_OVS_SLAVE_TYPE=ovs-bridge
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' ovs-bridge '!=' ovs-bridge ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f connection.master conn show 1e7cf87e-a6c0-467c-ae88-70fb0e0d701b
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + BRIDGE_NAME=br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' br-ex '!=' br-ex ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + CONFIGURATION_FILE=/run/ofport_requests.br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + declare -A INTERFACES
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' -f /run/ofport_requests.br-ex ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ cat /run/ofport_requests.br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + echo 'Sourcing configuration file '\''/run/ofport_requests.br-ex'\'' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: Sourcing configuration file '/run/ofport_requests.br-ex' with contents: declare -A INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53" )
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + source /run/ofport_requests.br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ INTERFACES=([eth2]="54" [eth1]="55" [eth5]="56" [eth4]="57" [vlan10]="53")
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ declare -A INTERFACES
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' a ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + ovs-vsctl set Interface vlan10 ofport_request=53
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + declare -p INTERFACES
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=br-ex2
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="br-ex2" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' '' == '' ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + exit 0
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + INTERFACE_NAME=br-ex
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + OPERATION=pre-up
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:18:50 rhel84 nm-dispatcher[7391]: ++ awk -F : '{if($1=="br-ex" && $2!~/^ovs*/) print $NF}'
Feb 08 11:18:51 rhel84 nm-dispatcher[7391]: + INTERFACE_CONNECTION_UUID=
Feb 08 11:18:51 rhel84 nm-dispatcher[7391]: + '[' '' == '' ']'
Feb 08 11:18:51 rhel84 nm-dispatcher[7391]: + exit 0
Feb 08 11:21:03 rhel84 dbus-daemon[774]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.549' (uid=0 pid=7408 comm="/usr/sbin/NetworkManager --no-daemon " label="system_u:system_r:NetworkManager_t:s0")
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_NAME=eth10
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + OPERATION=pre-up
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ awk -F : '{if($1=="eth10" && $2!~/^ovs*/) print $NF}'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_CONNECTION_UUID=923b22f9-8cc8-32fa-aa0d-54cffb024215
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' 923b22f9-8cc8-32fa-aa0d-54cffb024215 == '' ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ nmcli -t -f connection.slave-type conn show 923b22f9-8cc8-32fa-aa0d-54cffb024215
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ awk -F : '{print $NF}'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' '' '!=' ovs-port ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + exit 0
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_NAME=eth8
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + OPERATION=pre-up
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ awk -F : '{if($1=="eth8" && $2!~/^ovs*/) print $NF}'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_CONNECTION_UUID=2b1f9ef1-5fb3-35f5-9269-5ab7ef70f5c9
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' 2b1f9ef1-5fb3-35f5-9269-5ab7ef70f5c9 == '' ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ nmcli -t -f connection.slave-type conn show 2b1f9ef1-5fb3-35f5-9269-5ab7ef70f5c9
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ awk -F : '{print $NF}'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' '' '!=' ovs-port ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + exit 0
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + [[ OVNKubernetes != \O\V\N\K\u\b\e\r\n\e\t\e\s ]]
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_NAME=eth9
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + OPERATION=pre-up
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' pre-up '!=' pre-up ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ nmcli -t -f device,type,uuid conn
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ awk -F : '{if($1=="eth9" && $2!~/^ovs*/) print $NF}'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_CONNECTION_UUID=9432aa2c-d1aa-30a1-b852-970bc72b687d
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' 9432aa2c-d1aa-30a1-b852-970bc72b687d == '' ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ nmcli -t -f connection.slave-type conn show 9432aa2c-d1aa-30a1-b852-970bc72b687d
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: ++ awk -F : '{print $NF}'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + INTERFACE_OVS_SLAVE_TYPE=
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + '[' '' '!=' ovs-port ']'
Feb 08 11:21:03 rhel84 nm-dispatcher[7977]: + exit 0

[root@rhel84 pre-up.d]# ovs-ofctl show br-ex
OFPT_FEATURES_REPLY (xid=0x2): dpid:0000caab9438e140
n_tables:254, n_buffers:0
capabilities: FLOW_STATS TABLE_STATS PORT_STATS QUEUE_STATS ARP_MATCH_IP
actions: output enqueue set_vlan_vid set_vlan_pcp strip_vlan mod_dl_src mod_dl_dst mod_nw_src mod_nw_dst mod_nw_tos mod_tp_src mod_tp_dst
 32(patch-to-br-int): addr:4e:62:7d:8c:d4:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 53(vlan10): addr:52:54:00:33:bc:f8
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 54(eth2): addr:52:54:00:94:f8:58
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 55(eth1): addr:52:54:00:68:cf:ce
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 56(eth5): addr:52:54:00:15:ec:29
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 57(eth4): addr:52:54:00:df:9c:35
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
 LOCAL(br-ex): addr:ca:ab:94:38:e1:40
     config:     0
     state:      0
     speed: 0 Mbps now, 0 Mbps max
OFPT_GET_CONFIG_REPLY (xid=0x4): frags=normal miss_send_len=0
[root@rhel84 pre-up.d]# 
~~~

Reply:
When the node reboots, `/run` is wiped anyway, so the directory should be in a good state thereafter.

Reply:
Ok, I guess I got confused because on bash clobber is enabled by default while in zsh is not.

Reply:
I didn't even know that clobber / noclobber was a thing. *shame*

I changed the line to:
~~~
declare -p INTERFACES >| $CONFIGURATION_FILE
~~~